config/main.py

```diff
@@ -73,16 +70,17 @@ def _change_bgp_session_status_by_addr(ipaddress, status, verbose):
     config_db = ConfigDBConnector()
     config_db.connect()
 
-    # Convert the IP address to uppercase because IPv6 addresses are stored
-    # in ConfigDB with all uppercase alphabet characters
-    config_db.mod_entry('bgp_neighbor', ipaddress.upper(), {'admin_status': status})
+    config_db.mod_entry('bgp_neighbor', ipaddress, {'admin_status': status})
```

Comment:
You will either need to add a `.lower()` call on ipaddress here or on line 95 (`_change_bgp_session_status_by_addr(ip_addr.lower(), status, verbose)`)

However, since we will only need to covert the IP address to lowercase when getting or modifying ConfigDB entries, I feel it is best to add the `.lower()` calls when passing into the config_db methods (i.e., simply change my two `.upper()` calls to `.lower()` calls).